docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -112,65 +112,43 @@ image::images/timestamp-override.png[]
-
-If you are cloning prebuilt {ml} rules to generate alerts for the new {ml} jobs,
-the following rules are affected:
-
-* <<unusual-linux-network-port-activity>>: Use
-`v2_linux_anomalous_network_port_activity_ecs` instead of
-`linux_anomalous_network_port_activity_ecs`.
-* <<anomalous-process-for-a-linux-population>>: Use
-`v2_linux_anomalous_process_all_hosts_ecs` instead of
-`linux_anomalous_process_all_hosts_ecs`.
-* <<unusual-linux-username>>: Use `v2_linux_anomalous_user_name_ecs` instead of
-`linux_anomalous_user_name_ecs`.
-* <<unusual-linux-process-calling-the-metadata-service>>: Use
-`v2_linux_rare_metadata_process` instead of `linux_rare_metadata_process`.
-* <<unusual-linux-user-calling-the-metadata-service>>: Use
-`v2_linux_rare_metadata_user` instead of `linux_rare_metadata_user`.
-* <<unusual-process-for-a-linux-host>>: Use `v2_rare_process_by_host_linux_ecs`
-instead of `rare_process_by_host_linux_ecs`.
-* <<unusual-process-for-a-windows-host>>: Use
-`v2_rare_process_by_host_windows_ecs` instead of
-`rare_process_by_host_windows_ecs`.
-* <<unusual-windows-network-activity>>: Use
-`v2_windows_anomalous_network_activity_ecs` instead of
-`windows_anomalous_network_activity_ecs`.
-* <<unusual-windows-path-activity>>: Use `v2_windows_anomalous_path_activity_ecs`
-instead of `windows_anomalous_path_activity_ecs`.
-* <<anomalous-windows-process-creation>>: Use
-`v2_windows_anomalous_process_creation` instead of
-`windows_anomalous_process_creation`.
-* <<anomalous-process-for-a-windows-population>>: Use
-`v2_windows_anomalous_process_all_hosts_ecs` instead of
-`windows_anomalous_process_all_hosts_ecs`.
-* <<unusual-windows-username>>: Use `v2_windows_anomalous_user_name_ecs` instead
-of `windows_anomalous_user_name_ecs`.
-* <<unusual-windows-process-calling-the-metadata-service>>: Use
-`v2_windows_rare_metadata_process` instead of `windows_rare_metadata_process`.
-* <<unusual-windows-user-calling-the-metadata-service>>: Use
-`v2_windows_rare_metadata_user` instead of `windows_rare_metadata_user`.
+{ml-cap} detection rules use {ml} jobs that have dependencies on data fields populated by the {beats} and {agent} integrations. In version 8.3, new {ml} jobs (prefixed with `V3`) were released to operate on the ECS fields available at that time. If you're using multiple versions of {beats} or {agent} (from both before and after 8.3), you may need to duplicate or create new {ml} rules that specify the new ML `V3` jobs and enable them to run alongside your existing rules, to ensure continued coverage from rules currently using `V1` or `V2` jobs.
+
+* If you only have 8.2 or earlier versions of {beats} and {agent}, you can continue using the same `V1` and `V2` {ml} jobs and the existing prebuilt {ml} rules. 
+
+* If you only have 8.3 or later versions of {beats} and {agent}, you can use the latest `V3` {ml} jobs. To generate alerts for anomalies using these jobs, duplicate the existing {ml} rules and reconfigure them to use the `V3` {ml} jobs.
```

Comment:
So in this instance, they actually don't need to duplicate the existing rules since the new rule updates will contain the change to point to the v3 jobs. All they'll need to do is load /update to the latest rules in the UI, and start using the v3 jobs/rules and they'll be 👍